randovania/data/json_data/prime1/Phendrana Drifts.txt

```diff
@@ -607,9 +607,13 @@ Asset id: 1880625556
                   Grapple Beam or Slope Jump (Expert)
               Grapple Beam and Movement (Intermediate)
   > Event - Thardus
-      Any of the following:
-          Thermal Visor
-          Combat (Beginner) and Invisible Objects (Beginner)
+      All of the following:
+          Thermal Visor or Invisible Objects (Beginner)
+          Any of the following:
+              Combat (Intermediate) or Normal Damage ≥ 300
+              All of the following:
+                  Combat (Beginner)
+                  Charge Beam or Morph Ball
```

Comment:
Why morph? Wouldn't it make sense to have morph and bombs?

Reply:
I was thinking about avoiding damage. You can't always avoid damage without morph.

Reply:
Makes sense